src/api/hub.ts

```diff
@@ -0,0 +1,31 @@
+import axios from 'axios';
+
+export default class HubGroupAPI {
+  private readonly defaultBaseUrl = 'https://63d6be0e-bd49-4c76-8fe5-195bb7cf88a5.mock.pstmn.io';
+  private readonly baseUrl: string;
+
+  private readonly apiPath = '/ent/api/templates/bundlegroups';
+
+  constructor(baseUrl?: string) {
+    this.baseUrl = baseUrl || this.defaultBaseUrl;
+  }
+
+  async callApi(uri: string): Promise<any[]> {
+    const response = await axios(`${this.baseUrl}${uri}`);
+    const { data } = response;
+
+    return data;
+  }
+
+  getHubGroups(): Promise<any[]> {
```

Comment:
1) getBundleGroups
2) This should return BundleGroup[]

Reply:
type `BundleGroup` has been created, I have created `BundleGroupId` as well for items with only `BundleGroupName` and `BundleGroupVersionId` properties (in this method for instance)